pydabs/src/pydabs/main.py

```diff
@@ -5,7 +5,9 @@
 
 def main():
     # Process command-line arguments
-    parser = argparse.ArgumentParser(description="Databricks job with catalog and schema parameters")
+    parser = argparse.ArgumentParser(
+        description="Databricks job with catalog and schema parameters"
+    )
```

Comment:
@pietern @kanterov It seems our templates and the ruff check in this repository are not yet in a consistent state? The PR check was failing before I manually reformatted this file in this commit.